### PR TITLE
HADOOP-15184. Add GitHub pull request template.

### PR DESCRIPTION
**Pull request by aajisaka**
JIRA: https://issues.apache.org/jira/browse/HADOOP-15184

* Updated the URL of the wiki page.

### CODE REVIEW

**Comment by steveloughran** [comment]
+1

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## NOTICE
+
+Please create a issue in ASF JIRA before opening a pull request,
```
Comment (adamantal) [inline .github/pull_request_template.md]:
```suggestion
Please create an issue in ASF JIRA before opening a pull request,
```

Comment (aajisaka) [reply .github/pull_request_template.md]: Thanks! Fixed.

**Comment by aajisaka** [comment]
@steveloughran @adamantal @aajisaka 
Thank you for your review!

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 76 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1171 | trunk passed |
| +1 | compile | 1035 | trunk passed |
| +1 | mvnsite | 932 | trunk passed |
| +1 | shadedclient | 3903 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 351 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 1224 | the patch passed |
| +1 | compile | 995 | the patch passed |
| +1 | javac | 995 | the patch passed |
| +1 | mvnsite | 879 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 2 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 743 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 356 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 974 | root in the patch failed. |
| +1 | asflicense | 43 | The patch does not generate ASF License warnings. |
| | | 9704 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.security.TestFixKerberosTicketOrder |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.7 Server=18.09.7 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1419/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1419 |
| Optional Tests | dupname asflicense mvnsite compile javac javadoc mvninstall unit shadedclient xml |
| uname | Linux 53090f7d33fa 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 524b553 |
| Default Java | 1.8.0_222 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1419/2/artifact/out/patch-unit-root.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1419/2/testReport/ |
| Max. process+thread count | 1344 (vs. ulimit of 5500) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1419/2/console |
| versions | git=2.7.4 maven=3.3.9 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**